docs/lib/analytics/fragments/flutter/getting-started/30_initAnalytics.md

```diff
@@ -29,7 +29,12 @@ class MyAmplifyApp extends StatefulWidget {
 
         Amplify.addPlugins([AmplifyAuthCognito(), AmplifyAnalyticsPinpoint()]);
 
-        Amplify.configure(amplifyConfig); 
+        try {
+            await Amplify.configure(amplifyconfig);      
+        } on AmplifyAlreadyConfiguredException {
+            print(
+                'Amplify was already configured. Looks like app restarted on android.');
```

Comment:
Hm, this doesn't look like the ideal customer experience. Do we have a way to make iOS & Android look the same to the customer? How can I help?

Message-wise, we should capitalize the `a` in `Android`. Here's a suggestion for an alternative phrasing:

```suggestion
                'Tried to reconfigure Amplify; this can occur when your app restarts on Android.');
```

Same comment for the other related occurrence(s) in this PR, too.

Reply:
There is an Android specific error due to Flutter.  For some reason Flutter Android does: 'Activity destroyed MainActivity' while the same thing doesn't really happen on iOS.  This can cause the Flutter initState method to be called multiple times, which would then cause an already configured exception.  

I will rename the text as you suggested.  Not too sure if there's anything you can do on your side to fix this. 

@Amplifiyer @jamesonwilliams